code/create_db2.sql

```diff
@@ -0,0 +1,33 @@
+drop table if EXISTS rotten_tomatoes_movies;
+
+-- Búðu til töfluna fyrir CSV gögnin
+CREATE TABLE IF NOT EXISTS rotten_tomatoes_movies (
+    rotten_tomatoes_link TEXT,
+    movie_title TEXT,
+    movie_info TEXT,
+    critics_consensus TEXT,
+    content_rating TEXT,
+    genres TEXT,
+    directors TEXT,
+    authors TEXT,
+    actors TEXT,
+    original_release_date TEXT,
+    streaming_release_date TEXT,
```

Comment:
Þetta er óviðeigandi gagnatýpa:
```suggestion
original_release_date DATE,
streaming_release_date DATE,
```